packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -157,6 +157,7 @@ export interface ILocalFluidDataStoreContextProps extends IFluidDataStoreContext
 	/**
 	 * @deprecated 0.16 Issue #1635, #3631
 	 */
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Too many effects of switching to `unknown`? It's internal so we should be able to do it.

Reply:
Oh, actually I missed that this was internal. Some of the related ones were alpha/legacy (I think). I'll take a stab at this.

Reply:
Turns out this (and another) were completely unused. So I just removed them altogether.